liblangutil/SemVerHandler.h

```diff
@@ -22,15 +22,15 @@
 
 #pragma once
 
-#include <libsolidity/parsing/Token.h>
+#include <liblangutil/Token.h>
 #include <string>
 #include <vector>
 
-namespace dev
-{
-namespace solidity
+namespace langutil
 {
 
+using ::langutil::Token;
```

Comment:
This should have no effect now and should be deleted.

Reply:
Ok - how about now in b42446 ?